homeassistant/components/tplink/config_flow.py

```diff
@@ -129,6 +288,55 @@ async def async_step_pick_device(
             data_schema=vol.Schema({vol.Required(CONF_DEVICE): vol.In(devices_name)}),
         )
 
+    async def async_step_pick_device_auth_confirm(
```

Comment:
This looks pretty much the same as `async_step_user_auth_confirm`, could we achieve what we want using only one of them? Less code = less tests to write & maintain ;-)

Reply:
Done